shell/browser/native_window_mac.mm

```diff
@@ -1451,7 +1474,14 @@ void ViewDidMoveToSuperview(NSView* self, SEL _cmd) {
 
     [effect_view setAutoresizingMask:NSViewWidthSizable | NSViewHeightSizable];
     [effect_view setBlendingMode:NSVisualEffectBlendingModeBehindWindow];
-    [effect_view setState:NSVisualEffectStateFollowsWindowActiveState];
+
+    if (visual_effect_state_ == VisualEffectState::ACTIVE) {
```

Comment:
Can we make this a switch so the compiler yells at you when you miss one of the enum options

Reply:
(Can be follow up to avoid blocking the minor)